e2e_tests/integration/bolt.spec.ts

```diff
@@ -20,7 +20,7 @@
 
 import { isEnterpriseEdition } from '../support/utils'
 
-/* global Cypress, cy, test, expect, before */
+/* global Cypress, cy, before */
```

Comment:
Do we need these comments? Can't it be set in the tsconfig.json for cypress?

Reply:
We could clean up making type definitions available, these are from before we had a tsconfig, the edits I made here are just to fix tslint warnings which caught that some imported types were unused